cmd/fs-v1-multipart.go

```diff
@@ -683,8 +683,8 @@ func (fs *FSObjects) CompleteMultipartUpload(ctx context.Context, bucket string,
 	}
 
 	// Deny if WORM is enabled
-	if globalWORMEnabled {
-		if _, err = fsStatFile(ctx, pathJoin(fs.fsPath, bucket, object)); err == nil {
+	if retention, isWORMBucket := isWORMEnabled(bucket); isWORMBucket {
```

Comment:
Why is this *after* we write the metafile? Don't we overwrite it?

Reply:
the meta data of multipart completion is written in multipart directory which is OK to be done before finalizing the object creation in main name space.